Podfile.lock

```diff
@@ -115,4 +115,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: dcce34aea2296a4c3e247b2b2ff8a0e9a1cce2b9
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```

Comment:
Is there a reason Cocoapods got downgraded?

Reply:
no reason.  I haven't dared to change my config for fear of more complications.  I will upgrade in the next PR.